examples/examples_nodejs_test.go

```diff
@@ -47,6 +47,22 @@ func TestAccCluster(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestAccKubernetesServiceIPv4RangeForCluster(t *testing.T) {
```

Comment:
Now that we have Python and .NET hooked up in CI, it'd be nice to add tests for those languages as well (and Go coming soon). I can help with that as a follow-up, if you'd like.

Reply:
I will follow up with these when I get a local dev setup working for EKS in these langs